views/userman.profile.dt

```diff
@@ -0,0 +1,34 @@
+extends layout
+
+block title
+	- auto title = "Edit profile";
+
+block body
+
+	.inputForm
+		h1 Personal information
+		form(action="profile", method="POST")
+			- if (useUserNames)
+				p
+					label(for="user_name") User name
+					input(type="text", name="user_name", readOnly, value=user.name)
```

Comment:
Not that it matters much due to its read-only state, but this should be just `name="name"` according to the userman implementation (it allows changing the user name in theory, but there is currently a bug in the web code).